blocks/inspector-controls/range-control/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function RangeControl( { label, value, instanceId, onChange, ...attributes } ) {
+	const id = 'inspector-range-control-' + instanceId;
+
+	return (
+        <div className="blocks-inspector-control">
```

Comment:
this breaks our css guidelines https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#css

The className here should be "blocks-range-control" and the subclassNames "blocks-range-control__label". This forces us to think in shared components instead of shared classNames. We could probably create a generic `BaseControl` responsible of showing the label and the surroundings.